MVVM/ViewModel/RoverMode_UIOverlay.cs

```diff
@@ -8,6 +8,8 @@ namespace RoverControlApp.MVVM.ViewModel;
 
 public partial class RoverMode_UIOverlay : UIOverlay
 {
+	[Export]
+	Label SafeModeIndicator;
```

Comment:
Naming convention

Reply:
Adressed in [1c7a46c](https://github.com/wisniax/RoverControlApp/pull/16/commits/1c7a46caf7707993795a390e68fa60fa18eff9c3)